src/access/tests/TestAccessControl.cpp

```diff
@@ -1437,61 +1449,59 @@ void TestAclValidateTarget(nlTestSuite * inSuite, void * inContext)
         {
             for (auto deviceType : validDeviceTypes)
             {
-                NL_TEST_ASSERT(inSuite,
-                               entry.SetTarget(0,
-                                               { .flags      = Target::kCluster | Target::kEndpoint | Target::kDeviceType,
-                                                 .cluster    = cluster,
-                                                 .endpoint   = endpoint,
-                                                 .deviceType = deviceType }) == CHIP_NO_ERROR);
-                NL_TEST_ASSERT(inSuite, accessControl.UpdateEntry(0, entry) != CHIP_NO_ERROR);
-                NL_TEST_ASSERT(inSuite, accessControl.CreateEntry(nullptr, entry) != CHIP_NO_ERROR);
+                EXPECT_TRUE(entry.SetTarget(0,
+                                            { .flags      = Target::kCluster | Target::kEndpoint | Target::kDeviceType,
+                                              .cluster    = cluster,
+                                              .endpoint   = endpoint,
+                                              .deviceType = deviceType }) == CHIP_NO_ERROR);
+                EXPECT_TRUE(accessControl.UpdateEntry(0, entry) != CHIP_NO_ERROR);
+                EXPECT_TRUE(accessControl.CreateEntry(nullptr, entry) != CHIP_NO_ERROR);
                 accessControl.DeleteEntry(1);
             }
         }
     }
 
     // Cannot target none
     {
-        NL_TEST_ASSERT(inSuite, entry.SetTarget(0, { .flags = 0 }) == CHIP_NO_ERROR);
-        NL_TEST_ASSERT(inSuite, accessControl.UpdateEntry(0, entry) != CHIP_NO_ERROR);
-        NL_TEST_ASSERT(inSuite, accessControl.CreateEntry(nullptr, entry) != CHIP_NO_ERROR);
+        EXPECT_TRUE(entry.SetTarget(0, { .flags = 0 }) == CHIP_NO_ERROR);
```

Comment:
Suggest all the EXPECT_TRUE(xxx == CHIP_NO_ERROR) be replaced with EXPECT_EQ(xxx, CHIP_NO_ERROR) and EXPECT_NEQ(xxx, CHIP_NO_ERROR)

Reply:
Fixed